scripts.js

```diff
@@ -71,7 +71,7 @@ $(document).ready(() => {
 
       // Loop through every category and write mentors to the page
       Object.entries(categoryMentors)
-        .sort((a, b) => a[1].length < b[1].length)
```

Comment:
I think we would like descending order. should this be flipped to b - a instead?

Reply:
Sounds good! I've updated the PR and screenshots. Ready for another round of review @jwu910